### PR TITLE
added support for package install/uninstall options

### PR DESCRIPTION
**Pull request by exodusftw**

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fixtures:
+  forge_modules:
+    stdlib: "puppetlabs/stdlib"
```
Comment (crayfishx) [inline .fixtures.yml]: Why is this needed?

Comment (exodusftw) [inline .fixtures.yml]: Do you mean .fixtures.yml in general, or just the stdlib module? If there's no stdlib usage here, we can definitely just scrap that

Comment (crayfishx) [inline .fixtures.yml]: Yep, was more referring to the stdlib inclusion - the rest looks good, thanks!

Comment (exodusftw) [inline .fixtures.yml]: andddddd.......it's gone

**Comment by crayfishx** [comment]
LGTM - I'll release this as 1.0.0 - no need to keep this module in 0.x

**Comment by exodusftw** [comment]
thanks!

**Comment by crayfishx** [comment]
@exodusftw it's released on the forge now....

**Comment by exodusftw** [comment]
thank you - in my Puppetfile already ;)
